rocky/rocky/views/scan_profile.py

```diff
@@ -19,6 +20,12 @@ class ScanProfileDetailView(FormView, OOIDetailView):
     def get_context_data(self, **kwargs) -> dict[str, Any]:
         context = super().get_context_data(**kwargs)
         context["mandatory_fields"] = get_mandatory_fields(self.request)
+        context["scan_profile_user"] = False
```

Comment:
`{% if scan_profile_user %}` also works if the variable does not exist and will just evaluate to False, so it is not necessary to add this.

```suggestion
```